laravel/database/migrations/2023_01_10_050834_create_signature_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateSignatureTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('signatures', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('contractID');
+            $table->text('slug');
+            $table->enum('status', ['sent', 'opened', 'signed']);
+            $table->timestamps(); // Use timestamps instead of individual date columns
```

Comment:
The signatures table should be keeping track of the application ID and user ID the document is associated with